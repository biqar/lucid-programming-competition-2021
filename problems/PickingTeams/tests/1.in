7
+ Alice Bob
+ Charlie David
+ Bob Charlie
? Alice Charlie
+ Emily Alice
? Emily Bob
? Emily David
