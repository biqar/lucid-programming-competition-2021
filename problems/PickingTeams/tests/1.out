yes
yes
no
2